EOS-uiux-Solutions/user-story
# Added delete and edit feature in comments section.

Signed-off-by: Hrishikesh Agarwal <hrishikeshagarwal@Hrishikeshs-Air.mshome.net>

**Issue Number**

fixes #



**Describe the changes you've made**

Added feature for deleting and editing a existing comment and reply.


**Test plan **

A good test plan should give instructions that someone else can easily follow.

1. Visit a story.
2. Add a comment to that story.
3. You can press delete and edit icon to test.



**Checklist**
